Importing excel files: ClassNotFoundException

If you follow the documentation (https://neo4j-contrib.github.io/neo4j-apoc-procedures/#_loading_excel_xls) and install the dependencies listed there,
you won't be able to import xlsx files like this:

```
call apoc.load.xls("/data/Book1.xlsx", 'Sheet1',{}) yield map 
return map
```

The invocation produces:
```
Neo.ClientError.Procedure.ProcedureCallFailed: Failed to invoke procedure `apoc.load.xls`: Caused by: java.lang.ClassNotFoundException: org.apache.commons.collections4.ListValuedMap
```

Interestingly enough, loading the same data but in xls format (as opposed to xlsx) will work. 
Looks like the missing dependency (Apache commons-collections) is required only for xlsx.

To reproduce, I use the following image:
```
FROM neo4j:3.4

ENV NEO4J_apoc_import_file_enabled=true NEO4J_apoc_export_file_enabled=true NEO4J_AUTH=none

RUN    apk add --no-cache --virtual .bootstrap-deps wget ca-certificates \
    && for x in \
          http://repo1.maven.org/maven2/org/apache/poi/poi/3.17/poi-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml/3.17/poi-ooxml-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml-schemas/3.17/poi-ooxml-schemas-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/xmlbeans/xmlbeans/2.6.0/xmlbeans-2.6.0.jar \
          http://repo1.maven.org/maven2/com/github/virtuald/curvesapi/1.04/curvesapi-1.04.jar \
          https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.4.0.1/apoc-3.4.0.1-all.jar \
       ; do wget $x -P /plugins; done \
    && apk del .bootstrap-deps
```
Build the following image:
```
docker build -t test:commons .
```

Run the container and mount a volume (so you can put yout test excel file there):
```
docker run --name=neo4j --publish=7474:7474 --publish=7687:7687 --volume=c:\\test:/data test:commons
```

Put the test file to the folder (c:\test in my case), open the browser and run:
```
call apoc.load.xls("/data/Book1.xlsx", 'Sheet1',{}) yield map 
return map
```


Adding the missing library:
```
FROM neo4j:3.4

ENV NEO4J_apoc_import_file_enabled=true NEO4J_apoc_export_file_enabled=true NEO4J_AUTH=none

RUN    apk add --no-cache --virtual .bootstrap-deps wget ca-certificates \
    && for x in \
          http://repo1.maven.org/maven2/org/apache/poi/poi/3.17/poi-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml/3.17/poi-ooxml-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml-schemas/3.17/poi-ooxml-schemas-3.17.jar \
          http://repo1.maven.org/maven2/org/apache/xmlbeans/xmlbeans/2.6.0/xmlbeans-2.6.0.jar \
          http://repo1.maven.org/maven2/com/github/virtuald/curvesapi/1.04/curvesapi-1.04.jar \
          http://repo1.maven.org/maven2/org/apache/commons/commons-collections4/4.1/commons-collections4-4.1.jar \
          https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/3.4.0.1/apoc-3.4.0.1-all.jar \
       ; do wget $x -P /plugins; done \
    && apk del .bootstrap-deps
```

solves the problem.

--- a/docs/loadcsv.adoc
+++ b/docs/loadcsv.adoc
@@ -150,6 +150,7 @@ For XLS files:
 
 Additional for XLSX files:
 
+* http://repo1.maven.org/maven2/org/apache/commons/commons-collections4/4.1/commons-collections4-4.1.jar[commons-collections4-4.1.jar^]
 * http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml/3.17/poi-ooxml-3.17.jar[poi-ooxml-3.17.jar^]
 * http://repo1.maven.org/maven2/org/apache/poi/poi-ooxml-schemas/3.17/poi-ooxml-schemas-3.17.jar[poi-ooxml-schemas-3.17.jar^]
 * http://repo1.maven.org/maven2/org/apache/xmlbeans/xmlbeans/2.6.0/xmlbeans-2.6.0.jar[xmlbeans-2.6.0.jar^]

